azure-mgmt-batchai/src/main/java/com/microsoft/azure/management/batchai/BatchAIFileServer.java

```diff
@@ -96,10 +96,18 @@
         interface WithGroup extends GroupableResource.DefinitionStages.WithGroup<WithDataDisks> {
         }
 
+        /**
+         * This stage of a Batch AI file server definition allows to specify data disks parameters.
+         */
         interface WithDataDisks {
             WithVMSize withDataDisks(int diskSizeInGB, int diskCount, StorageAccountType storageAccountType);
+
+            WithVMSize withDataDisks(int diskSizeInGB, int diskCount, StorageAccountType storageAccountType, CachingType cachingType);
```

Comment:
should the method name be `withDataDisk` (without "s")

Reply:
We provide diskCount parameter, which means that more than 1 data disk can be attached. From the diskCount parameter description:  'Number of data disks to be attached to the VM. RAID level 0 will be applied in the case of multiple disks.'